eclipsesource/cdtcloud-deploymentserver
# feat: Search in workspace, syntax highlighting, react fix

## Changes Proposed:


- Pin a react setup that doesn't mess with `@theia/core` and/or hoisting
- Add Cpp syntax highlighting

## How to Test the Changes:


1. `yarn install && yarn run download:plugins`
2. `docker-compose up`
3. `yarn run dev`
4. Open Arduino workspace, see syntax highlighting in .ino files
5. Open console, then cdtcloud widget, notice a lack of complaints
